store/sqlstore/command_store.go

```diff
@@ -91,54 +96,56 @@ func (s SqlCommandStore) GetByTrigger(teamId string, trigger string) (*model.Com
 		query = "SELECT * FROM Commands WHERE TeamId = :TeamId AND \"trigger\" = :Trigger AND DeleteAt = 0"
 	}
 
-	if err := s.GetReplica().SelectOne(&command, query, map[string]interface{}{"TeamId": teamId, "Trigger": trigger}); err != nil {
-		return nil, model.NewAppError("SqlCommandStore.GetByTrigger", "store.sql_command.get_by_trigger.app_error", nil, "teamId="+teamId+", trigger="+trigger+", err="+err.Error(), http.StatusInternalServerError)
+	if err := s.GetReplica().SelectOne(&command, query, map[string]interface{}{"TeamId": teamId, "Trigger": trigger}); err == sql.ErrNoRows {
+		return nil, store.NewErrNotFound("Command", trigger)
+	} else if err != nil {
+		return nil, errors.Wrapf(err, "selectbytrigger: team_id=%s, trigger=%s", teamId, trigger)
 	}
 
 	return &command, nil
 }
 
-func (s SqlCommandStore) Delete(commandId string, time int64) *model.AppError {
+func (s SqlCommandStore) Delete(commandId string, time int64) error {
 	_, err := s.GetMaster().Exec("Update Commands SET DeleteAt = :DeleteAt, UpdateAt = :UpdateAt WHERE Id = :Id", map[string]interface{}{"DeleteAt": time, "UpdateAt": time, "Id": commandId})
 	if err != nil {
-		return model.NewAppError("SqlCommandStore.Delete", "store.sql_command.save.delete.app_error", nil, "id="+commandId+", err="+err.Error(), http.StatusInternalServerError)
+		errors.Wrapf(err, "delete: command_id=%s", commandId)
 	}
 
 	return nil
 }
 
-func (s SqlCommandStore) PermanentDeleteByTeam(teamId string) *model.AppError {
+func (s SqlCommandStore) PermanentDeleteByTeam(teamId string) error {
 	_, err := s.GetMaster().Exec("DELETE FROM Commands WHERE TeamId = :TeamId", map[string]interface{}{"TeamId": teamId})
 	if err != nil {
-		return model.NewAppError("SqlCommandStore.DeleteByTeam", "store.sql_command.save.delete_perm.app_error", nil, "id="+teamId+", err="+err.Error(), http.StatusInternalServerError)
+		return store.NewErrInvalidInput("Command", "TeamId", teamId)
```

Comment:
We are not differentiating between invalid teamId and an actual error in the sql command. It may not be ideal, but I'd like to keep that consistency in this PR. That just means we can use `errors.Wrapf` to wrap and return.